src/applications/mhv/secure-messaging/api/mocks/allrecipients/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
+const allRecipients = {
+  data: [
+    {
+      id: '1013155',
+      type: 'triage_teams',
+      attributes: {
+        triageTeamId: 1013155,
+        name: '***MEDICATION_AWARENESS_100% @ MOH_DAYT29',
+        stationNumber: '989',
+        blockedStatus: false,
+        preferredTeam: true,
+        relationshipType: 'PATIENT',
+      },
+    },
+    {
+      id: '2710520',
+      type: 'triage_teams_test',
+      attributes: {
+        triageTeamId: 2710520,
+        name: 'SM_TO_VA_GOV_TRIAGE_GROUP_TEST _ OLD',
+        stationNumber: '110',
+        blockedStatus: false,
+        preferredTeam: true,
+        relationshipType: 'PATIENT',
+      },
+    },
+  ],
+  meta: {
+    associatedTriageGroups: 1,
```

Comment:
this value would be 2 since 2 triage teams are included in a response
```suggestion
    associatedTriageGroups: 2,
```